doc/interop-test-descriptions.md

```diff
@@ -206,25 +245,75 @@ Client asserts:
 * call was successful
 * response aggregated_payload_size is 74922
 
+
+### client_compressed_streaming
+
+This test verifies the client can compress streaming messages. It sends an
+initial inconsistent streaming call comprised of a single message to verify if
+the server implements the [CompressedRequest][] feature. If it does, the client
+will then send another streaming call, comprised of two messages: the first one
+compressed with `expect_compressed` true; the second one uncompressed with
+`expected_compressed` false.
+
+Procedure:
+ 1. Client calls StreamingInputCall
+ 1. Client sends the following feature-probing *uncompressed* message
```

Comment:
That's a feature. Markdown will do the right thing when it renders it. By having always 1., we don't need to change all the numbers if we insert something in the middle, etc.